bakery/README.md

```diff
@@ -139,7 +139,7 @@ Note: This is probably most useful for the `cops-bakery-scripts` image, but you
 #### Tip for development, deployment, and QA
 
 Both the Bakery CLI and the Pipeline Config Generator allow you to specify a tag of a remote image to use with the `--tag`. For example, if a tag, `important-tag`, has been released for each of our images, one can:
-1. Generate a pipeline pinning all versions of images to that tag with usage like `./build pipeline pdf staging --tag=important-tag`
+1. Generate a pipeline pinning all versions of images to that tag with usage like `./build pipeline cops staging --tag=important-tag`
```

Comment:
Minor: In addition to the README.md, we may want to make corresponding updates of `pdf` --> `cops` to the Sphinx docs (e.g. in [updating the stack](https://github.com/openstax/output-producer-service/blob/master/docs/operations/updating_the_stack.rst)).

Reply:
good catch